cfme/services/catalogs/ansible_catalog_item.py

```diff
@@ -191,15 +192,20 @@ class retirement(Tab):  # noqa
         info = SummaryForm("Retirement Info")
         variables_and_default_values = Table(".//div[@id='retirement']//table")
 
+
+class DetailsAnsibleCatalogItemView(ServicesCatalogView):
+    """Has to be in view standards, changed for WidgetasticTaggable.get_tags()"""
+    entities = View.nested(DetailsEntitiesAnsibleCatalogItemView)
```

Comment:
This ```DetailsEntitiesAnsibleCatalogItemView``` and ```entities```   has been added due to the fact that we have to use [PR5296#L296](https://github.com/mshriver/integration_tests/blob/e966eb6aa9bbac1d26c97e84f4219662411f99c5/cfme/common/__init__.py#L296) for get_tags() from new WidgetasticTaggable mixins